Samples/Shared/Config/Signing.xcconfig

```diff
@@ -5,9 +5,9 @@ CODE_SIGN_IDENTITY_Debug = iPhone Developer
 CODE_SIGN_IDENTITY_Test = iPhone Developer
 CODE_SIGN_IDENTITY_TestCI = iPhone Developer
 CODE_SIGN_IDENTITY_Release = iPhone Distribution
-CODE_SIGN_IDENTITY_Debug[sdk=macosx*] = Apple Developer
-CODE_SIGN_IDENTITY_Test[sdk=macosx*] = Apple Developer
-CODE_SIGN_IDENTITY_TestCI[sdk=macosx*] = Apple Developer
+CODE_SIGN_IDENTITY_Debug[sdk=macosx*] = Apple Development
```

Comment:
`m`: How did you test this change?

Reply:
It was breaking somewhere in CI in with the old value; `Apple Developer` is not a valid signing identity designator for mac apps; `Apple Development` is the correct form.